### PR TITLE
cmplog enhancement

### PR DESCRIPTION
**Pull request by vanhauser-thc**

### CODE REVIEW

File: instrumentation/afl-compiler-rt.o.c [hunk 1]
```diff
@@ -154,6 +158,12 @@ static void at_exit(int signal) {
 
 }
 
+#ifdef WORD_SIZE_64
+  #define default_hash(a, b) XXH64(a, b, HASH_CONST)
+#else
+  #define default_hash(a, b) XXH64(a, b, HASH_CONST)
```
Comment (devnexen) [inline instrumentation/afl-compiler-rt.o.c]: did you mean something else ?

Comment (vanhauser-thc) [reply instrumentation/afl-compiler-rt.o.c]: no, why do you ask? afl-compiler-rt has `#include "xxhash.h"`

Comment (devnexen) [reply instrumentation/afl-compiler-rt.o.c]: I meant `default_hash` is the same for both cases unless I missed something.

Comment (vanhauser-thc) [reply instrumentation/afl-compiler-rt.o.c]: :D ah yes you are right. I thought we have a different one for 32 bit, but we have not. ty!

**Comment by vanhauser-thc** [comment]
the other variant will be merged.

**Pull request closed without merging**